PWGEM/PhotonMeson/TableProducer/createPCM.cxx

```diff
@@ -554,15 +554,15 @@ struct createPCM {
   PROCESS_SWITCH(createPCM, processTrkCollAsso, "create V0s with track-to-collision associator", false);
 };
 
-// Extends the v0data table with expression columns
-struct v0Initializer {
-  Spawns<aod::V0Cores> v0cores;
-  void init(InitContext const&) {}
-};
+// // Extends the v0data table with expression columns
+// struct v0Initializer {
+//   Spawns<aod::V0Cores> v0cores;
+//   void init(InitContext const&) {}
+// };
```

Comment:
I guess instead of commenting this can also just be removed (git will tell us the history if needed :-) )